src/main/service/mediaAccountApplicationService.ts

```diff
@@ -0,0 +1,58 @@
+import {AccountManagementRequests} from '../resources/requests/accountManagementRequests';
+import {CreateAccountService} from '../service/createAccountService';
+import {MediaAccount} from '../models/mediaAccount';
+import moment from 'moment';
+
+const accountManagementRequests = new AccountManagementRequests();
+const createAccountService = new CreateAccountService();
+
+export class MediaAccountApplicationService {
+
+  public async getApplicationById(applicationId): Promise<MediaAccount | null> {
+    if (applicationId) {
+      const mediaAccount = await accountManagementRequests.getMediaApplicationById(applicationId);
+      if (mediaAccount) {
+        mediaAccount.requestDate = moment(Date.parse(mediaAccount.requestDate)).format('DD MMMM YYYY');
+        return mediaAccount;
+      }
+    }
+
+    return null;
+  }
+
+  public async getApplicationByIdAndStatus(applicationId, status): Promise<MediaAccount | null> {
+    const mediaAccount = await this.getApplicationById(applicationId);
+
+    if (mediaAccount && mediaAccount.status === status) {
+      return mediaAccount;
+    }
+
+    return null;
+  }
+
+  public async getApplicationImageById(imageId): Promise<Blob> {
```

Comment:
is this getting an application or is this getting an image?

Reply:
It's getting an image.  I've updated this now to getImageById.